README.md

```diff
@@ -1,13 +1,120 @@
+money in ERTP, we withdraw `payments` from purses. 
+
+```js
+const paymentForAlice = purse.withdraw(10, `alice's community money`);
+```
+
+Like our purse, this payment contains BaytownBucks, but unlike purses,
+payments are used to represent tokens in transit. A payment can be
+sent to someone else, a purse never should be. 
+
+Now let's send the payment to Alice as message:
+
+```
+alice.receivePayment(paymentForAlice);
+```
+
+This may seem strange, but ERTP is built on top of [an
+infrastructure](https://github.com/Agoric/SwingSet) in which
+everything is an object. In this example, we have a reference to the
+object `alice`, and can call her `receivePayment` method to send her a
+payment. Alice's methods are entirely up to her, and are not part of
+ERTP.
+
+## Security Properties
+
+How does Alice know that she got paid real money? She could have been
+sent fake money, or she could have been sent money that was
+[double-spent](https://en.wikipedia.org/wiki/Double-spending). 
+
+When alice receives an alleged payment, she can call a method to know
+that the payment is 1) valid, and 2) exclusively hers:
+
+```js
+const myExclusivePayment = BaytownBucksIssuer.claimAll(allegedPayment);
+```
+
+The BaytownBucksIssuer is associated with the BaytownBucksMint, but
+the issuer is the public-facing version that is accessible to anyone.
+By holding the reference to a mint, you can mint more tokens. By
+holding a reference to the issuer for a mint, you can check that a
+payment is valid and exclusively claim it for yourself. 
+
+That's the basic use case for a fungible token. `makeMint` in
+[issuers.js](core/issuers.js) takes
+in an optional configuration that allows for many more possibilities. 
+
+## Pixel Gallery Demo
+
+To explore further types of digital assets, beyond fungible tokens, we
+decided to create a demo for our
+[testnet](https://github.com/Agoric/cosmic-swingset) in which users
+can buy and sell the right to color a pixel on a webpage. Read more
+[here](/GALLERY-README.md).
+
+## Gotchas
+
+When looking at the code in our [tests](test), you might see some new
+concepts:
+
+* __Vats__: All user code runs in what we call a `vat`. Within a vat,
+  code is run synchronously. Communication with another vat happens
+  asynchronously. The [SwingSet
+  infrastructure](https://github.com/Agoric/SwingSet) creates the vats
+  and makes communication between vats possible. 
+
+* __E() and infix bang (!)__: With our Swingset infrastructure, we can
+  talk to objects in another vat as if they were local, which is
+  really cool! In order to do so, we need something more than the
+  "dot" syntax. So instead of `obj.foo()`, we either do `E(obj).foo()` or
+  the syntactic sugar, `obj!foo()`.
+
+* __Presences__: Presences are our name for the local object that
+  stands in for the remote object. So when we say `obj!foo()`, `obj`
+  is a presence because it represents a remote object that the message
+  `foo` will be passed to.
+
```

Comment:
> So when we say `obj!foo()`, `obj` is a presence because it represents a remote object that the message `foo` will be passed to.

This makes it sound like we only use the syntax above when `obj` is a presence. In fact, `obj` might just be a local object, a local promise, a local presence for a remote object, or a remote promise. The key thing is that all of these designate some actual object --- though it may be separated in space and time. The syntax means "deliver the message `foo()` to the actual object asynchronously, in its own turn, wherever and whenever it is, even if it is local."

Reply:
Good point. I added your language to this section and the E/! section. let me know if it still needs work!

Reply:
> So when we say `obj!foo()` and intend to send a message to a remote object, obj is a presence because it represents a remote object that the message foo will be passed to.

This phrasing is better but it still has the same problem. Maybe first state that `obj` is a presence of a remote object, and so serves as its local ambassador, representative, mascot, or something. (Don't say "proxy"!) Then explain that this enables us to send messages to the remote object by using "!" on the local presence of that remote object.

Reply:
Ok, I changed it to this

Reply:
Perfect.